import-automation/import-progress-dashboard-frontend/README.md

```diff
@@ -0,0 +1,17 @@
+# Import Progress Dashboard Frontend
+
+This directory contains code for the frontend for the Import Progress Dashboard.
+The frontend has two components: a server and a client. The client is a single
+page application that displays a table of system runs and the server serves the
+web page and retrieves data from the Import Progress Dashboard API for the
+client.
+
+## Deploying to App Engine
+
+```
+cd client
```

Comment:
Could you do it as:
(cd client; npm install && ￼npm run-script build)

Reply:
I kind of want it to be clear that there are several steps involved